backend/graphql/OpenUniversityRegistrationLink.ts

```diff
@@ -19,6 +19,7 @@ export const OpenUniversityRegistrationLink = objectType({
     t.model.course_code()
     t.model.language()
     t.model.link()
+    t.model.tiers()
```

Comment:
This might have to be replaced with a field resolver if Nexus gets mixed up with Json array

Reply:
I've replaced `t.model.tiers()` with a field resolver with a Json type (looking at UserCourseProgress.ts and UserCourseSetting.ts as examples). Please check if I have done this correctly.